cmd/kafka-sampler/README.md

```diff
@@ -78,16 +83,16 @@ Kafka supports many authentication methods, since its configuration is not strai
 
 | Config file YAML key              | Env var                            | Value       |
 |-----------------------------------|------------------------------------|-------------|
-| `kafka.sarama.net.sasl.enable`    | `KAFKA_SARAMA_NET_SASL_ENABLE`     | `true`      |
+| `kafka.sarama.net.sasl.enable`    | `KAFKA_SARAMA_NET_SASL_ENABLE`     | *true*      |
 | `kafka.sarama.net.sasl.user`      | `KAFKA_SARAMA_NET_SASL_USER`       | `<username>`|
 | `kafka.sarama.net.sasl.password`  | `KAFKA_SARAMA_NET_SASL_PASSWORD`   | `<password>`|
 
 ##### SASL/SCRAM
 
 | Config file YAML key              | Env var                            | Value                               |
 |-----------------------------------|------------------------------------|-------------------------------------|
-| `kafka.sarama.net.sasl.enable`    | `KAFKA_SARAMA_NET_SASL_ENABLE`     | `true`                              |
-| `kafka.sarama.net.sasl.mechanism` | `KAFKA_SARAMA_NET_SASL_MECHANISM`  | `SCRAM-SHA-256` or `SCRAM-SHA-512`  |
+| `kafka.sarama.net.sasl.enable`    | `KAFKA_SARAMA_NET_SASL_ENABLE`     | *true*                              |
+| `kafka.sarama.net.sasl.mechanism` | `KAFKA_SARAMA_NET_SASL_MECHANISM`  | `SCRAM-SHA-256* or *SCRAM-SHA-512`  |
```

Comment:
Is this change expected?

Reply:
no, I applied a search and replace and seems this was unintentionally affected